### DISCUSSION TITLE
Can you make a "add template" option in setting

### DISCUSSION THREAD

**Discussion by syedsimanta03** [felladrin/linkedin-autoconnect-chrome-extension#34] [Ideas]
### Is your feature request related to a problem? Please describe.

I don't get add a note when sending request

### Describe the solution you'd like

I think it should take dynamic first name {{firstname}} and send the pre saved note

**Comment by felladrin** [comment]
That's a good idea! I want to try it when I have some free time.
I'll move it to the Discussions section, meanwhile. Thanks!
